packages/web3-eth/src/web3_subscriptions.ts

```diff
@@ -77,6 +96,35 @@ export class NewPendingTransactionsSubscription extends Web3Subscription<
 	}
 }
 
+/**
+ * ## subscribe(“newHeads”)
```

Comment:
From my understanding `subscribe("newHeads")` and subscribe("newBlockHeaders")` is the same, can we mention that? As well show the example of `web3.eth.subscribe("newHeads")`